app/temporal/src/domain/DataAnalyzer.spec.ts

```diff
@@ -410,13 +416,23 @@ describe("DataAnalyzer", () => {
     it("should validate multiple validations", () => {
       const rowsWithDuplicateValues = [{ __rowId: 0 }, { __rowId: 1 }];
       const validatorColumns = {
-        required: [{ column: "name" }],
-        unique: [{ column: "name" }],
-        phone: [{ column: "name" }],
-        email: [{ column: "name" }],
-        regex: [{ column: "name", regex: "^[0-9]{5}$" }],
-      } as Record<ValidatorType, { column: string; regex?: string }[]>;
-      const stats: Stats = { name: { nonunique: { undefined: 2 } } };
+        required: [{ column: "name", config: { type: "required" } }],
+        unique: [{ column: "name", config: { type: "unique" } }],
+        phone: [{ column: "name", config: { type: "phone" } }],
+        email: [{ column: "name", config: { type: "email" } }],
+        regex: [
+          {
+            column: "name",
+            config: {
+              type: "regex",
+              regex: "^[0-9]{5}$",
+            } as RegexColumnValidation,
+          },
+        ],
```

Comment:
kannst du den enum valdiator hier noch hinzufügen, dann haben wir in dem testcase alle